array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,9 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  if (array.length === 0) return [];
```

Comment:
This isn't actually necessary. If the array length is `0`, then your loop will never start, so you end up with the same result